Aptoide/aptoide-client-v8
# [ASV-1872] New deeplink for editorial

**What does this PR do?**

   This PR aims at implementing the deeplink for editorial, coming from the web. Deeplink is on the ticket.
Also, took the chance to fix the title of editorial, since the apps with longer titles were getting cut.

**Database changed?**

 No

**Where should the reviewer start?**


**How should this be manually tested?**

  Send the deeplink and check that it opens the page correctly.
Also test that the previous deeplink is also working and that both requests are working fine (loading by card id and by slug).

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1872](https://aptoide.atlassian.net/browse/ASV-1872)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
